src/modules/aws_ecr/index.ts

```diff
@@ -158,9 +156,10 @@ class RepositoryImageMapper extends MapperBase<RepositoryImage> {
           const idRegion = decoded[4];
           if (enabledRegions.includes(idRegion)) {
             const client = (await ctx.getAwsClient(idRegion)) as AWS;
-            const rawImage = await this.getRepositoryImage(client.ecrClient, [imageId], decoded[3]);
-            if (rawImage?.images && rawImage.images[0]) {
-              return await this.repositoryImageMapper(rawImage.images[0], ctx, type, idRegion);
+            const rawImage = await this.listRepositoryImages(client.ecrClient, [imageId], decoded[3]);
```

Comment:
Maybe this `decoded` variable can be replaced by using `this.idFields` function? Or can be done as a follow-up refactor task.

Reply:
seems like more of a nit, can be a follow-up. I think it is okay as-is